### PR TITLE
Option to transfer ETH to bridge canister on deploy

### PR DESCRIPTION
**Pull request by Maximkaaa**
## Issue ticket

Issue ticket link: https://infinityswap.atlassian.net/browse/EPROD-1051

### CODE REVIEW

File: src/bridge-deployer/src/commands/mod.rs [hunk 1]
```diff
@@ -206,23 +209,30 @@ impl Bridge {
...
                 };
 
-                let btf_address = btf_args
-                    .deploy_btf(network, bridge_principal, pk, agent, false, evm)
+                let contracts = btf_args
+                    .deploy_btf(network.clone(), bridge_principal, pk, agent, false, evm)
                     .await?;
 
-                info!("Base BTF bridge deployed with address {btf_address:?}");
+                info!("Base BFT bridge deployed with address {contracts:?}");
```
Comment (ufoscout) [inline src/bridge-deployer/src/commands/mod.rs:216]: BFT -> BTF

**Review by ufoscout — CHANGES_REQUESTED** [review]

File: src/bridge-deployer/src/contracts.rs [hunk 2]
```diff
@@ -332,26 +335,65 @@ impl SolidityContractDeployer<'_> {
...
+            .get_transaction_count(address, BlockNumber::Latest)
+            .await?;
+        let chain_id = client.get_chain_id().await?;
+
+        let tx = TransactionBuilder {
+            from: &address.into(),
+            to: Some((*to).into()),
+            nonce: nonce.into(),
+            value: amount.into(),
+            gas: 5_000_000u64.into(),
```
Comment (veeso) [inline src/bridge-deployer/src/contracts.rs:385]: Isn't it 5M a lot?

**Review by veeso — COMMENTED** [review]
I just have a doubt on the gas field and whether we can have tests for this function

File: src/bridge-deployer/src/contracts.rs (hunk 2 above)
Comment (Maximkaaa) [reply src/bridge-deployer/src/contracts.rs:385]: As far as I understand thsi option is just the limit we are willing to spend on the transaction. Actual value used will be calculated by the network.

**Review by veeso — APPROVED** [review]

**Review by F3kilo — APPROVED** [review]

**Pull request merged**